stream-helpers/map.ts

```diff
@@ -0,0 +1,33 @@
+import type { Operation, Stream } from "effection";
+
+/**
+ * Transforms each item in the stream using the provided function.
+ *
+ * @param fn - The function to transform each item
+ * @returns A stream transformer that applies the function to each item
+ */
+export function map<A, B, TClose = never>(
```

Comment:
`map()` shouldn't be parameterized with `TClose` since it can be applied to any stream and you don't need to know the full type of the stream ahead of time in order to create the map operation (only that it contains items of type `A`).

Instead, the type parameter should be moved to the _returned_ function. That way it can be inferred whenever it is applied.

Reply:
Ok, done